### PR TITLE
21 - Comments list

### PR DESCRIPTION
**Pull request by mar7in91Wolox**
# Summary
- I added the comments into the Details View

# Trello Card
https://trello.com/c/wDurFbUk

# Screenshots
image

### CODE REVIEW

File: training-net/src/Controllers/MovieController.cs [hunk 1]
```diff
@@ -17,13 +18,22 @@ namespace training_net.Controllers
...
-        public MovieController(IUnitOfWork unitOfWork) => this._unitOfWork = unitOfWork;
-
-        public IUnitOfWork UnitOfWork
+        private readonly UserManager<User> _userManager;
+        public MovieController(IUnitOfWork unitOfWork, UserManager<User> userManager)
+        {
+            this._unitOfWork = unitOfWork;
+            this._userManager = userManager;
+        }
+        private IUnitOfWork UnitOfWork
```
Comment (truccomarcos) [inline training-net/src/Controllers/MovieController.cs]: Don't forget to separate this methods

File: training-net/src/Controllers/MovieController.cs [hunk 2]
```diff
@@ -17,13 +18,22 @@ namespace training_net.Controllers
     public class MovieController : Controller
     {
         private readonly IUnitOfWork _unitOfWork;
-        public MovieController(IUnitOfWork unitOfWork) => this._unitOfWork = unitOfWork;
-
-        public IUnitOfWork UnitOfWork
+        private readonly UserManager<User> _userManager;
```
Comment (truccomarcos) [inline training-net/src/Controllers/MovieController.cs]: Add an empty line to separate the attributes from the constructor

File: training-net/src/Controllers/MovieController.cs (hunk 1 above)
Comment (mar7in91Wolox) [reply training-net/src/Controllers/MovieController.cs]: Done

File: training-net/src/Controllers/MovieController.cs (hunk 2 above)
Comment (mar7in91Wolox) [reply training-net/src/Controllers/MovieController.cs]: Done!

**Pull request merged**